engine/graphics/src/null/graphics_null.cpp

```diff
@@ -75,6 +76,7 @@ namespace dmGraphics
         m_Height                  = params.m_Height;
         m_Window                  = params.m_Window;
         m_PrintDeviceInfo         = params.m_PrintDeviceInfo;
+        m_JobThread               = params.m_JobThread;
```

Comment:
The null adapter can now use threads if we want to. This is mostly used for testing right now, the reason being I wanted to make sure the job system API works. But maybe we don't want to do this change at all? I.e skip the test(s) in test_graphics and instead do some sort of runtime validation as we do for OpenGL?